### PR TITLE
guardian doc for admins

### PR DESCRIPTION
**Pull request by angelaLazar**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/multifactor-authentication/guardian/admin-guide.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Users that were previously registered before you enable MFA, will complete the same process as new users on their next login.
+
+By enabling **Push Notifications** for MFA, you will also be enabling the option to use Google Authenticator instead of Guardian. To use this option, your users must have the Google Authenticator app downloaded to their mobile device, the app is available for both Android and iOS. When a new user signs up, or a user is logging in for the first time since enabling MFA there will be a code to be scanned in the Google Authenticator app. After scanning the code, they will get a six digit code to enter. Once they enter this code, there will be a confirmation screen which has a recovery code. They need to have this recovery code to login without their mobile device. If they lose both the recovery code and their mobile device, you will need to reset their MFA [see below](/admin-guide#reset-mfa-for-a-user) on how to do this.
```
Comment (dafortune) [inline articles/multifactor-authentication/guardian/admin-guide.md]: > They need to have this recovery code to login without their mobile device. If they lose both the recovery code and their mobile device, you will need to reset their MFA [see below](/admin-guide#reset-mfa-for-a-user) on how to do this.

I know the process is the same, and someone might have a different opinion, but it sounded kind of repetitive when I read it first time.

**Comment by dafortune** [comment]
Just a comment. Looks good to me

**Comment by dafortune** [comment]
Would make sense to include some screenshots from Guardian dashboard?

**Comment by woloski** [comment]
I dont see any mention about SMS

**Comment by dafortune** [comment]
What about Guardian audit logs? I think we should mention them

File: articles/multifactor-authentication/guardian/admin-guide.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+You may choose to enable Guardian only for select users. Within the Customize MFA code snippet, you can include conditions to enable Guardian.
+
+For example, suppose you want to *omit* MFA for all users signing in from the `foo.com` domain.
+
+ ```js
+ function (user, context, callback) {
+ 
+     if (context.connection !== 'foo.com'){
+         context.multifactor = {
+             provider: 'guardian', //required
```
Comment (hzalaz) [inline articles/multifactor-authentication/guardian/admin-guide.md]: We should always suggest to add `ignoreCookies: true` in the multi factor options

Comment (woloski) [inline articles/multifactor-authentication/guardian/admin-guide.md]: And also explain what it means to set this

**Comment by angelaLazar** [comment]
I added a section on SMS, audit events, updateCookies and added more screenshots

File: articles/multifactor-authentication/guardian/admin-guide.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+Users that were previously registered before you enable MFA, will complete the same process as new users on their next login.
+
+By enabling **Push Notifications** for MFA, you will also be enabling the option to use Google Authenticator instead of Guardian. To use this option, your users must have the Google Authenticator app downloaded to their mobile device, the app is available for both Android and iOS. When a new user signs up, or a user is logging in for the first time since enabling MFA there will be a code to be scanned in the Google Authenticator app. After scanning the code, they will get a six digit code to enter. Once they enter this code, there will be a confirmation screen which has a recovery code. They need enter this recovery code to login without their mobile device. If they do not have the recovery code or their mobile device, you will need to [reset MFA](/admin-guide#reset-mfa-for-a-user) for their account.
```
Comment (woloski) [inline articles/multifactor-authentication/guardian/admin-guide.md]: This paragraph is too complex and should be somwhere else, not on Guardian intro. Like a note saying that Guardian behind the scenes uses the same mechanism that Google Authenticator (or any TOTP Authenticator) uses, so for phones where there is no Guardian (e.g. Windows Phones), you can scan the QR code using Microsoft Authenticator. Actually, I would make a special section explaining the support on different types of devices and versions of OS.

File: articles/multifactor-authentication/guardian/admin-guide.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+ * **Friendly Name**: the name of the app that you want displayed to the users
+ * **Logo URL**: the URL that points to the logo image you want displayed to your users
+
+Auth0 recommends using a logo image that is at least 100x100 pixels, though an image that is 200x200 pixels ensures quality viewing in devices with Retina or high DPI displays.
+
+## Tracking your Users MFA Events 
+
+In the [Logs](${uiURL}/#/logs) section of the dashboard you can see the various events related to your users signing up and signing in using MFA.
+
+![](/media/articles/mfa/logs.png)
```
Comment (woloski) [inline articles/multifactor-authentication/guardian/admin-guide.md:93]: I don't see Guardian Logs in this screen

Comment (angelaLazar) [inline articles/multifactor-authentication/guardian/admin-guide.md:93]: I updated the screenshot to have more guardian events

**Comment by angelaLazar** [comment]
closing in favor of: https://github.com/auth0/docs/pull/1398

**Pull request closed without merging**